src/scripts.js

```diff
@@ -223,7 +223,7 @@ function all(userData, userRepo, user, activityData, activity, activityRepo, dat
           }
         }
       });
-    $('.longest-sleepers').text(`${findUserName(sleepRepo.returnWeeklyLongestSleepers(1)[1])}: ${sleepRepo.returnWeeklyLongestSleepers(1)[0]} hours`);
+    $('.longest-sleepers').text(`${findUserName(sleepRepo.returnWeeklyLongestSleepers(1)[1])}: ${sleepRepo.returnWeeklyLongestSleepers(1)[0].toFixed(2)} hours`);
```

Comment:
Just checking, looks like jquery lets you do this without putting Number. first 

https://stackoverflow.com/questions/22367752/jquery-tofixed2-fixed-number-reduce-to-3-digits

Reply:
@ckoga explained to me that doing it this way makes it into a string.  Since it's just for display, sounds like that's fine.